server.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+
+const app = express();
+const PORT = 3000;
+
+app.use(express.static(__dirname + "/dist"));
+
+const navLinks = [
+  "/",
+  "/login",
+  "/signup",
+  "/profile",
+  "/changepass",
+  "/500",
+  "/incorrectName",
+];
+
+navLinks.forEach((link) => {
+  app.get(link, (req, res) => {
```

Comment:
Здесь можно
```suggestion
  app.get('/*', (req, res) => {
```
Чтобы отобразить страницу 404 при неправильном урле